akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -72,6 +72,9 @@ sealed trait HttpEntity extends jm.HttpEntity {
       .via(new akka.http.impl.util.ToStrict(timeout, contentType))
       .runWith(Sink.head)
 
+  def discardEntityBytes(mat: Materializer): HttpMessage.DiscardedEntity =
```

Comment:
We're not changing any names, we're adding a new one, and that can have the short name because we're already _in the entity_:

```scala
request.entity.discardBytes() // new method
request.discardEntityBytes() // currently exists, no change
```